scipy/stats/tests/test_stats.py

```diff
@@ -1263,8 +1263,10 @@ def test_signaltonoise(self):
 
         # y = stats.signaltonoise(self.shoes[0])
         # assert_approx_equal(y,4.5709967)
-        y = stats.signaltonoise(self.testcase)
-        assert_approx_equal(y,2.236067977)
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", RuntimeWarning)
```

Comment:
The second argument is the warning category to filter, should be ``DeprecationWarning`` in this case.